.circleci/config.yml

```diff
@@ -149,19 +146,34 @@ jobs:
           command: xcodebuild test -workspace << parameters.workspace >>.xcworkspace -scheme << parameters.scheme >> -sdk iphonesimulator -destination "${destination}" | xcpretty --simple --color --report junit
       - store_test_results:
           path: build/reports
-  
+
   deploy:
     <<: *defaults
+    parameters:
+      lane:
+        type: string
+        default: "unstable"
```

Comment:
I guess we're all comfortable with the "unstable" token across the three repos? I would have preferred edge or next or something, but I didn't get a vote. I'm furious.